src/session/SessionMiddleware.ts

```diff
@@ -43,22 +41,21 @@ function sessionRequestHandler(config: CookieConfig, sessionStore: SessionStore)
 
             } catch (err) {
 
-                console.error(err);
+                loggerInstance().error(err);
                 response.clearCookie(config.cookieName);
                 delete request.session;
 
                 try {
                     const cookie = Cookie.createFrom(sessionCookie);
                     sessionStore.delete(cookie);
-                } catch (_) {
-                    console.error(_);
+                } catch (err) {
+                    loggerInstance().error(err);
                 }
 
             }
 
         } else {
-            console.log(`REQUEST: ${request.url}`);
-            console.log("No Session cookie.");
+            loggerInstance().info(`No session cookie - REQUEST: ${request.url}`);
```

Comment:
```suggestion
           loggerInstance().infoRequest(request);
           loggerInstance().debug(`No session cookie`);
```

Reply:
There should be only one log message produced in this instance. Otherwise it will be much harder to correlate logs in environments serving large number of users.